### ISSUE TITLE
Trojan detected

### ISSUE THREAD

**Issue by Amateur2** [hadley/devtools#1144]
When installing devtools (using RStudio) on a mac running the El Capitan OS,
my (Intego) Virus Barrier detected a Trojan: OSX/Flashback.G in openssl.so
I don't know whether the problem is with the package itself, or the server, or something else.
But I thought I should let you know.
If the problem is not with the package, but something else, perhaps you would be kind enough to re-direct this report to the right person(s)? Many thanks

**Comment by jeroenooms** [comment]
I'm 99% confident this is a false positive (lots of trojans use openssl code). The `openssl.so` file is part of the `openssl` package, so this is definitely the wrong repo. Do you get the same error when installing openssl?

```r
install.packages("openssl")
```

And what happens when you try installing openssl from source?

```r
install.packages("openssl", type = "source")
```

**Comment by Amateur2** [comment]
Thank you for responding so promptly.
I use Intego VirusBarrier8 (with up-to-date virus definitions) which I
understand to be reliable for use on macs (which is what I'm using), but
I'm by no means an expert.
When I ran a full scan subsequently, it picked up the same file
(openssl.so), and 'repaired' it when I selected that option.

I sent it to you because the error message popped up immediately after I
typed:

install.packages ("devtools")

into the RStudio screen.

It was the first instruction I'd typed, so I thought you were the obvious
people to send a message to, but  if I understand you correctly, it might
be another package used in the process rather than devtools itself - in
which case, my apologies.

As will be evident, I'm a novice so reluctant to tempt fate by opening
'openssl' again. However if you are unable to reproduce the problem, it
sounds as if it's ok for others to use, which was my motive in contacting
you.

Many thanks for your time, and for getting back to me so quickly.
